sci-libs/minc/minc-2.2.00.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Medical Imaging file format and Toolbox"
+HOMEPAGE="http://packages.bic.mni.mcgill.ca/tgz/${P}.tar.gz"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="BSD"
+
+DEPEND="sys-devel/flex
+	sys-devel/autoconf
+	sys-devel/gcc[cxx]
```

Comment:
If you don't check in the ebuild that g++ used, you can drop it from depend. I can have gcc[cxx] installed but having CXX=icpc defined.